src/ui/session/ViewSession.tsx

```diff
@@ -3,21 +3,31 @@ import { useNavigate } from 'react-router-dom';
 import { Screens } from '@/ui/App';
 import { SubjectType } from '@/model/Subject';
 import { Button } from '@wordpress/components';
+import { getSchemas } from '@/model/Schema';
+
+const schemas = getSchemas();
 
 export function ViewSession() {
 	const { session } = useSessionContext();
 	const navigate = useNavigate();
 
-	const importButtons = [
-		{
-			text: 'Import Posts',
-			url: Screens.blueprints.new( session.id, SubjectType.BlogPost ),
-		},
-		{
-			text: 'Import Pages',
-			url: Screens.importPagesStart( session.id ),
-		},
-	];
+	const importButtons: { text: string; url: string }[] = [];
+	Object.keys( schemas ).forEach( ( subjectType ) => {
+		// Pages get a specific button.
+		if ( subjectType === SubjectType.Page ) {
+			importButtons.push( {
+				text: `Import Pages`,
+				url: Screens.importPagesStart( session.id ),
+			} );
+			return;
+		}
```

Comment:
This is so that we maintain the behavior that was in place before this PR, which is that pages have a custom flow, where you manually import them one by one. I would like to not change this behavior in this PR, so that's why this code is here.

We can revisit this behavior in the future if there is a need for that.